input/pagecontent/terminology.md

```diff
@@ -26,44 +26,45 @@ See the [FHIR terminology section]({{site.data.fhir.path}}terminologies-valueset
 |[Clinical Condition](https://healthterminologies.gov.au/fhir/ValueSet/clinical-condition-1)|[AU Base Condition](StructureDefinition-au-condition.html)|NCTS|
 |[Clinical Finding](https://healthterminologies.gov.au/fhir/ValueSet/clinical-finding-1)|[AU Base Allergy Intolerance](StructureDefinition-au-allergyintolerance.html), [AU Base Condition](StructureDefinition-au-condition.html), [AU Base Diagnostic Imaging Report](StructureDefinition-au-imagingreport.html), [AU Base Diagnostic Report](StructureDefinition-au-diagnosticreport.html), [AU Base Dosage](StructureDefinition-au-dosage.html), [AU Base Pathology Report](StructureDefinition-au-pathologyreport.html)|NCTS|
 |[Clinical Specialty](https://healthterminologies.gov.au/fhir/ValueSet/clinical-specialty-1)|[AU Base Healthcare Service](StructureDefinition-au-healthcareservice.html), [AU Base Practitioner Role](StructureDefinition-au-practitionerrole.html)|NCTS|
-|[Common AU Recorded Sex or Gender (RSG) Source Document Jurisdiction](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-rsg-source-document-jurisdiction.html)|[AU Base Patient](StructureDefinition-au-patient.html), [AU Base Practitioner](StructureDefinition-au-practitioner.html), [AU Base Related Person](StructureDefinition-au-relatedperson.html)|AU Base|
-|[Common AU Recorded Sex or Gender (RSG) Source Document Type](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-rsg-source-document-type.html)|[AU Base Patient](StructureDefinition-au-patient.html), [AU Base Practitioner](StructureDefinition-au-practitioner.html), [AU Base Related Person](StructureDefinition-au-relatedperson.html)|AU Base|
-|[Common AU Recorded Sex or Gender Type](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-rsg-type.html)|[AU Base Patient](StructureDefinition-au-patient.html), [AU Base Practitioner](StructureDefinition-au-practitioner.html), [AU Base Related Person](StructureDefinition-au-relatedperson.html)|AU Base|
+|[AU Recorded Sex or Gender (RSG) Source Document Jurisdiction](ValueSet-rsg-source-document-jurisdiction.html)|[AU Base Patient](StructureDefinition-au-patient.html), [AU Base Practitioner](StructureDefinition-au-practitioner.html), [AU Base Related Person](StructureDefinition-au-relatedperson.html)|AU Base|
```

Comment:
The location of the gender and sex value sets including pronouns is their respective extensions not directly to an element in the profile..

Reply:
Noted - amended location to respective extension, thanks.